projects/core/src/cart/store/reducers/cart.reducer.ts

```diff
@@ -1,5 +1,5 @@
 import * as fromAction from './../actions';
-import { OrderEntry } from '@spartacus/core';
+import { OrderEntry } from '../../../occ-models/index';
 
```

Comment:
Please move selectors at the end of reducers to selectors (where memoized selectors are defined)